discrimination-ieee754/src/Data/Discrimination/IEEE754.hs

```diff
@@ -0,0 +1,46 @@
+-- | Discrimination of floating-point numbers via their IEEE754 representation.
+--
+-- This technique is described by http://stereopsis.com/radix.html.
+--
+-- This orders floating-point numbers according to the IEEE754 floating-point
+-- total order.  -NaN < -Infinity < ... < -0 < 0 < ... < +Infinity < +NaN.
+-- NaNs with positive sign are ordered by the value of their payload, and NaNs
+-- with negative sign are reverse-ordered by the value of their payload.
+
+module Data.Discrimination.IEEE754 where
```

Comment:
Use an explicit export list, and perhaps only export a subset of these functions (e.g. just the "group" and "sort" variants)?

Up to you.

Reply:
Done.